package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -116,6 +126,17 @@ <h2><%:Overview%></h2>
 					<dt><%:Firmware%></dt><dd><%| nodeinfo.software.firmware.release %></dd>
 					<% if nodeinfo.software.fastd then -%>
 						<dt><%:Mesh VPN%></dt><dd><%| enabled(nodeinfo.software.fastd.enabled) %></dd>
+						<% if nodeinfo.network.mesh_vpn and nodeinfo.network.mesh_vpn.bandwidth_limit.enabled then -%>
+						<dt><%:Bandwidth limit%></dt>
+						<dd>
+							<% if nodeinfo.network.mesh_vpn.bandwidth_limit.egress then -%>
+							▲ <%| formatBits(nodeinfo.network.mesh_vpn.bandwidth_limit.egress*1000) %>/s egress<br />
+							<%- end %>
```

Comment:
Still should be Upstream/Downstream for consistency. Other things look good to me (Just philosophical issues , but if it works, it works).

Reply:
Lets use the thread you already opened for this.
https://github.com/freifunk-gluon/gluon/pull/2217#discussion_r645780228